ouroboros-network/framework/lib/Ouroboros/Network/ConnectionManager/Types.hs

```diff
@@ -385,14 +384,6 @@ newtype ConnectionHandler muxMode handlerTrace socket peerAddr handle handleErro
       }
 
 
--- | Boolean like type
---
-data Inactive =
```

Comment:
Doesn't the removal of this type make it a breaking change?
It also doesn't seem related to the bug fix, at least break it out in a separate commit.